docs/source/optional_plugins/varianter_cit.rst

```diff
@@ -0,0 +1,129 @@
+====================
+CIT Varianter Plugin
+====================
+
+This plugin is an implementation of a Combinatorial Interaction
+Testing algorithm for the Avocado varianter functionality.  It
+generates an optimal number of variants, which in turn become
+different test scenarios.
+
+.. note:: The publication by Ahmed, Bestoun S., Kamal Z. Zamli, and
+          Chee Peng Lim, entitled “Application of particle swarm
+          optimization to uniform and variable strength covering array
+          construction”, Applied Soft Computing, 12(4), 2012,
+          pp. 1330-1347, contains the basis for the algorithm and
```

Comment:
Do we want to include link? I found (hopefully) it here: http://booksc.org/book/13151468/aa9fc9

Reply:
Yes, makes sense.